owned-manifests/clusterapi-apiserver-role-binding.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: RoleBinding
 metadata:
   name: clusterapi
-  namespace: {{ .TargetNamespace }}
+  namespace: kube-system
```

Comment:
@enxebre what's the `TargetNamespace` set to by default?

Reply:
> @enxebre what's the `TargetNamespace` set to by default?

Based on usage below in this file and what I saw, it is the namespace where you're creating a server.  Which is fine in general, but this resource belongs in a different namespace.